source/NVDAObjects/JAB/__init__.py

```diff
@@ -231,7 +231,7 @@ def getEmbeddedObject(self, offset=0):
 
 
 class JAB(Window):
-	def findOverlayClasses(self, clsList):
+	def findOverlayClasses(self, clsList) -> None:
```

Comment:
please add a type for clsList

```suggestion
	def findOverlayClasses(self, clsList: list[NVDAObject]) -> None:
```